### DISCUSSION TITLE
Looking for exploits for Read & Write

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1214] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
My school has the Read and Write extension but I don't know what useful scripts can be used with it can y'all give me a list of useful scripts?

**Comment by S-PScripts** [comment]
you can use scripts? where? i've downloaded the extension and can't see any.
as for the scripts you could use, that's up to you to think about.
